gridstatus/miso_api.py

```diff
@@ -1393,6 +1393,156 @@ def get_medium_term_load_forecast_daily(
             time_resolution=DAILY_RESOLUTION,
         )
 
+    @support_date_range(frequency="DAY_START")
+    def get_outage_forecast(
```

Comment:
Note: this method is not a replacement for `MISO().get_generation_outages_forecast()`. As far as we know from MISO, that report is not being retired. We need to add `get_outage_forecast` to replace `MISO().get_look_ahead_hourly` which is a combination of outage forecast and mid term load forecast and is being retired.

[Data Exchange Report to Endpoint Mapping726669.pdf](https://github.com/user-attachments/files/23736818/Data.Exchange.Report.to.Endpoint.Mapping726669.pdf)

<img width="1529" height="68" alt="Image" src="https://github.com/user-attachments/assets/245687ed-3f9b-4c55-a5a1-57825096213b" />